src/wallet/signer.rs

```diff
@@ -395,18 +394,18 @@ impl SignersContainer {
 
     /// Returns the list of signers in the container, sorted by lowest to highest `ordering`
     pub fn signers(&self) -> Vec<&Arc<dyn Signer>> {
-        self.0.values().collect()
+        let mut items = self.0.iter().collect::<Vec<_>>();
+        items.sort_by(|(a, _), (b, _)| (*a).cmp(*b));
```

Comment:
I think you could even use the `sort_unstable_by()` to squeeze out some extra performance

Reply:
Done. I also used the `_by_key` variant to make it more readable since the key implements comparison traits.